src/main/java/com/github/onsdigital/babbage/api/endpoint/content/MaxAge.java

```diff
@@ -54,6 +62,40 @@ protected boolean verifyKey(String key) {
     }
 
     protected int getMaxAge(HttpServletRequest request) throws Exception {
+        if (appConfig().babbage().isLegacyCacheAPIEnabled()) {
+            CloseableHttpClient httpClient = HttpClients.createDefault();
+            RequestBuilder requestBuilder = RequestBuilder.create(request.getMethod())
+                .setUri(appConfig().babbage().getLegacyCacheProxyUrl());
+
+            Enumeration<String> headerNames = request.getHeaderNames();
+            while (headerNames.hasMoreElements()) {
+                String headerName = headerNames.nextElement(); 
+                String headerValue = request.getHeader(headerName);
+                requestBuilder.addHeader(headerName, headerValue);
+            }
+
+            HttpUriRequest forwardRequest = requestBuilder.build();
+
+            CloseableHttpResponse proxyResponse = httpClient.execute(forwardRequest);
+
+             Header cacheControlHeader = proxyResponse.getFirstHeader("Cache-Control");
+
+             int maxAgeValue = 0;
+             if (cacheControlHeader != null) {
+                 String cacheControlValue = cacheControlHeader.getValue();
+                 String[] directives = cacheControlValue.split(",");
+                 for (String directive : directives) {
+                     if (directive.trim().startsWith("max-age=")) {
```

Comment:
Small one, but you could save the results of `directive.trim()` in a variable so that you don't have to do it twice.